app/services/stories_api/v3/endpoints/story.rb

```diff
@@ -64,6 +64,12 @@ def strip_admin_params
             @params[:story].delete(field) if @params[:story].present?
           end
         end
+
+        def story_params
+          return params[:story] if params.class == Hash
```

Comment:
I'm curious about this guard clause - aren't params always `ActionController::Parameters`?

Reply:
Sometimes the function is called but the params attribute is not a request parameters...

Reply:
Huh, okay. Thanks for the response